### ISSUE TITLE
Question: Trello vs GitHub Projects

### ISSUE THREAD

**Issue by stefanos82** [ClassicPress/ClassicPress#31]
@scottybo You mentioned in another [issue](https://github.com/ClassicPress/ClassicPress/issues/22#issuecomment-418034957) about covering financial expenses for services such as Trello.

What's wrong with Projects that come by default with each repository?

It's a kanban board, much like you get with Trello.

**Comment by nylen** [comment]
+1 for sticking with GitHub.  Adding Trello would have a number of disadvantages:

- Fragments our data and planning further
- Expenses
- Not as easy for casual users to browse, understand, and participate

**Comment by warrior79** [comment]
It's a good idea to use GitHub Projects, IMO what matters it's the Kanban Board approach

**Comment by Mte90** [comment]
We need only to structure the repositories in a way that they are created in the right repo like for proposal, bugs, expenses etc.

**Comment by scottybo** [comment]
Happy to go with projects - never used it before (nor did I know it even exists!)

**Comment by nylen** [comment]
We should also consider building our own small, custom tools based on GitHub. One example would be organizing and voting on feature proposals.

**Comment by stefanos82** [comment]
@nylen can you provide an example? Did you mean to extend GitHub in a way to meet our needs?

**Comment by nylen** [comment]
Yes.  For example, say that we want to use GitHub to track features that are proposed for inclusion in the project.  I think this is totally reasonable, and at first, we could just use GitHub issues and labels.

Then, when we know more about our specific needs beyond what GitHub offers, we can start building things out a bit more, without breaking the existing interactions via GitHub.  In this case the workflow could look something like this:

- Create an issue for each proposed change or feature
- Assign all of these issues the same label like `feature proposal`
- Provide a way for community members to indicate interest in each proposal (this can be as simple and natural as counting the number of issue comments and/or emoji reactions)
- Build a read-only web UI to display feature proposals and their possible decisions, sorted by popularity.  The UI will pull data from these same issues.
- Come up with some other mechanism (a specific comment or action by a committee member, perhaps) to track a decision, and indicate this in the web UI too.

Creating an initial version of a tool like this should take 5-10 hours - but I don't think we even need to go that far yet.

To me this is a better use of our resources than paying for a separate tool which then requires its own infrastructure, separate logins, etc.

**Comment by ginsterbusch** [comment]
@nylen I agree with that. Esp. when the next thing I'm going to work on is a port / partial fork of the Betatester Plugin, which in our case would have to pull in its data straight from Github.

**Comment by striebwj** [comment]
We can use Fider with a sign in option of Github to allow easy access to the platform without the need of two sign ins. This allows developers on Github to access it, while also ensuring end-users (non-developers) can also easily access it. 

I spent today setting it up, just need to point a domain to it then it is all good to go.

**Comment by nylen** [comment]
@striebwj I appreciate the effort, but in general I think we should be careful about adding a lot of new tools, especially before we know we have a need for them.

It's nice that Fider allows GitHub logins, but I am not crazy about fragmenting our discussion and planning even further (we already have GitHub and Slack).
